dev/scripts/src/alluxio.org/build/cmd/version.go

```diff
@@ -36,7 +32,11 @@ func VersionF() error {
 
 var versionRe = regexp.MustCompile(".*<version>(.*)</version>.*")
 
-func alluxioVersionFromPom(repoRoot string) (string, error) {
+func alluxioVersionFromPom() (string, error) {
```

Comment:
my original intention for providing the repoRoot as an argument is to minimize the calls to `findRepoRoot`. but seems like you are calling it in additional locations where it's not as easy to provide. okay to keep as is, but wanted to call it out

Reply:
worth using `sync.Once` to prevent it from being called so much?

Reply:
it'd need to be stored as a package level variable but that's fine for this specific case